apps/web/src/routes/_content/resources/newsletters/$newsletterName-$newsletterId.tsx

```diff
@@ -127,7 +143,7 @@ function NewsletterDetail() {
             paddingClass="p-5 md:p-[30px]"
           >
             <article className="w-full flex flex-col md:flex-row gap-5 lg:gap-9">
-              <div className="flex flex-col items-center justify-center">
+              <div className="flex flex-col">
                 <img
                   className="md:w-[367px] mx-auto object-cover rounded-[10px] lg:max-w-[347px] md:mx-0 lg:rounded-[22px] mb-8 lg:mb-12"
```

Comment:
Margin bottom under image is too big, should be mb-[30px] on desktop and mb-5 on mobile to harmonize with podcast